tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.pipeline.tesadapter.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TesAdapterController {
+
+    @GetMapping("/v1/tasks/service-info")
+    public ResponseEntity<String> serviceInfo(
+            @RequestParam(name = "status", required = false, defaultValue = "OK") String status){
```

Comment:
what is status? Please look at description of TES API:
https://github.com/ga4gh/task-execution-schemas/blob/master/task_execution.swagger.json
method `/v1/tasks/service-info` This is a method without any parameters, so it suppose to return status of service based on it own health (f.i. real service could check some metrics from Spring Actuator, but now we want just stub it, so here we need to return 'OK')

Reply:
fixed